### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by figmentbml**
some boring changes

### CODE REVIEW

File: write_ups/write_code_to_match_the_explanation.md [hunk 1]
```diff
@@ -4,66 +4,80 @@ Write code to match each of the following explanations.
...
 ```
 
 4) Provide a statement that concatenates the values `6` and `8`
 
-**Response:** 
+**Response:**
 
 ```
-# your code here
+"6" + "8"
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:33]: This was a trick question. You don't concatenate numbers.

File: write_ups/write_code_to_match_the_explanation.md [hunk 2]
```diff
@@ -4,66 +4,80 @@ Write code to match each of the following explanations.
...
 #### Revisiting Classes and Objects
 
 6) Evaluate the final statement from *Question #5*.
 
-**Response:** 
+**Response:**
 
 ```
-# your code here
+"211313 arbitrary"
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:53]: Can you show me where in the code this whitespace character comes from?

File: write_ups/write_code_to_match_the_explanation.md [hunk 3]
```diff
@@ -4,66 +4,80 @@ Write code to match each of the following explanations.
...
 
 7) Create a `Car` class, that is constructed with `make` and `model` attributes.
 
-**Response:** 
+**Response:**
 
 ```
-# your code here
+class Car
+  def initialize (make, model)
```
Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:62]: You're passing the arguments, but what do you need to... do with them?

File: write_ups/write_code_to_match_the_explanation.md (hunk 2 above)
Comment (figmentbml) [inline write_ups/write_code_to_match_the_explanation.md:53]: Yeah there isn't one

Comment (amk-boCO) [inline write_ups/write_code_to_match_the_explanation.md:53]: :)

**Pull request closed without merging**